docs/ref/validators.txt

```diff
@@ -285,3 +285,23 @@ to, or in lieu of custom ``field.clean()`` methods.
       ``decimal_places``.
     - ``'max_whole_digits'`` if the number of whole digits is larger than
       the difference between ``max_digits`` and ``decimal_places``.
+
+``FileExtensionValidator``
+--------------------------
+
+.. class:: FileExtensionValidator(allowed_extensions, message, code)
+
+    .. versionadded:: 1.10
+
+    Raises a :exc:`~django.core.exceptions.ValidationError` with a code of
```

Comment:
I think there should be a warning that the file extension of the uploaded object doesn't necessary correspond to the actual object (e.g. a user can rename a file to have any extension).